cmd/pqsd/main.go

```diff
@@ -57,6 +59,18 @@ func run(ctx context.Context) error {
 	opts := []pqstream.ServerOption{
 		pqstream.WithTableRegexp(tableRe),
 	}
+
+	if (len(*redactions)) > 0 {
+		rfields := make(pqstream.RedactionFields)
+		if err := json.NewDecoder(strings.NewReader(*redactions)).Decode(&rfields); err != nil {
+			log.Fatalf("Error decoding redactions. Make sure it's in valid JSON format i.e '{\"public\":{\"users\":[\"password\",\"ssn\"]}}'  Error: %v", err)
```

Comment:
follow error conventions of no upper-case. I think we can omit the example here and should prefer it in the field's help.

I'd prefer a simple:

    return errors.Wrap(err, "decoding redactions")